subsys/net/ip/l2/Kconfig

```diff
@@ -93,6 +93,8 @@ config NET_L2_BT_SHELL
 
 source "subsys/net/ip/l2/ethernet/Kconfig"
 
+source "subsys/net/ip/l2/gptp/Kconfig"
```

Comment:
no it's not, seriously. And actually we don't care about the lenght at all.

Actually, when gptp was first done, ethernet did not have its own directory. Thus why the original put it there, I guess. Now we have an ethernet dir, move it there.

Reply:
No, seriously it certainly matters. Not because of any technical reason but because it annoying to use very deep directories. As discussed offline, I will later propose that we move subsys/net/ip/l2 to subsys/net/l2 which is a more logical place for it and it will make the directory tree a bit shallower.